src/__tests__/starWarsSchema.js

```diff
@@ -145,29 +156,37 @@ const characterInterface = new GraphQLInterfaceType({
  *     name: String
  *     friends: [Character]
  *     appearsIn: [Episode]
+ *     homePlanet: String
  *     secretBackstory: String
  *   }
  */
+type HumanJS = {
+  id: string,
+  name?: string,
+  friends?: string[],
+  appearsIn?: number[],
+  homePlanet?: string,
+};
 const humanType = new GraphQLObjectType({
   name: 'Human',
   description: 'A humanoid creature in the Star Wars universe.',
   fields: () => ({
     id: {
-      type: new GraphQLNonNull(GraphQLString),
+      type: nonNull(GraphQLString),
```

Comment:
please leave these as they were - this additional change distracts from the purpose of the PR